projects/plugins/jetpack/_inc/client/components/section-nav/item.jsx

```diff
@@ -30,33 +31,46 @@ class NavItem extends React.PureComponent {
 		};
 		itemClasses[ 'dops-section-nav-' + itemClassPrefix ] = true;
 		const itemClassName = classNames( itemClasses );
-		let target, onClick;
-
-		if ( this.props.isExternalLink ) {
-			target = '_blank';
-		}
+		let onClick;
 
 		if ( ! this.props.disabled ) {
 			onClick = this.props.onClick;
 		}
 
 		return (
 			<li className={ itemClassName }>
-				<a
-					href={ this.props.path }
-					target={ target }
-					className={ 'dops-section-nav-' + itemClassPrefix + '__link' }
-					onClick={ onClick }
-					tabIndex={ this.props.tabIndex || 0 }
-					disabled={ this.props.disabled }
-					role="menuitem"
-					rel={ this.props.isExternalLink ? 'external' : null }
-				>
-					<span className={ 'dops-section-nav-' + itemClassPrefix + '__text' }>
-						{ this.props.children }
-						{ 'number' === typeof this.props.count && <Count count={ this.props.count } /> }
-					</span>
-				</a>
+				{ ! this.props.isExternalLink && (
```

Comment:
minor suggestion: You could use ternary condition:

```es6
{ ! this.props.isExternalLink
  ? ( ... )
  : ( ... )
}
```

Reply:
thanks!